code-samples/grant/node/update-grants/index.js

```diff
@@ -0,0 +1,103 @@
+import * as dotenv from "dotenv";
+import Nylas from "nylas";
+import inquirer from "inquirer";
+
+dotenv.config();
+
+const nylas = new Nylas({
+  apiKey: process.env.API_KEY,
+});
+
+const grants = await nylas.auth.grants.list();
+
+async function main() {
+  if (grants.data.length === 0) {
+    console.log("No grants found!");
+    return;
+  }
+
+  console.clear();
+  try {
+    const answer = await inquirer.prompt([
+      {
+        type: "list",
+        name: "grant",
+        message: "Please select an Gmail account to update:",
+        choices: grants.data.map(({ email, id }) => ({
+          name: email,
+
+          value: {
+            id,
+            email,
+          },
+        })),
+      },
+      {
+        type: "checkbox",
+        name: "scopes",
+        message: "Scopes:",
+        choices: [
+          {
+            name: "Gmail ReadOnly",
+            value: "gmail.readonly",
+          },
+          {
+            name: "Gmail Send",
+            value: "gmail.send",
+          },
+          {
+            name: "Gmail Modify",
+            value: "gmail.modify",
+          },
+          {
+            name: "Calendar",
+            value: "calendar",
+          },
+          {
+            name: "Contacts",
+            value: "contacts",
+          },
+        ],
+      },
+    ]);
+
+    const { requestId, data: grant } = await nylas.auth.grants.update({
+      grantId: answer.grant.id,
+      requestBody: {
+        scope: answer.scopes,
+      },
+    });
+
+    console.log("\x1b[32m%s\x1b[0m", "\n❯ Grant updated successfully!");
+    console.log(
+      "\x1b[32m%s\x1b[0m",
```

Comment:
@amit-ksh - what does `\x1b[32m%s\x1b[0m` represent?

Reply:
Makes the text green and bold.

Reply:
Oh that’s pretty cool!

Reply:
Should I remove it or add a comment? It might be confusing for the viewers.

Reply:
Another option could be to use a constant:

const boldTextWrapper = "\x1b[32m%s\x1b[0m"

and just re-using that.

Reply:
Another option could be to use a constant:

const boldTextWrapper = "\x1b[32m%s\x1b[0m"

and just re-using that.

Reply:
Done!